### PR TITLE
[expo-sensors] Fixed parity issues, added more information on functionality

### PR DESCRIPTION
**Pull request by EvanBacon**
# Why

iOS & Android function differently. This fixes those issues and makes it easier to test the Accelerometer going forward. 

#3117 

# How

* Inversed iOS values on Accelerometer
* Inversed Gravity on iOS & Web to match DeviceMotion across all platforms
* Updated docs with information about functionality regarding Gyroscope.

# Test Plan

Added a screen to NCL

### CODE REVIEW

File: ios/Exponent/Kernel/Services/EXSensorManager.m [hunk 1]
```diff
@@ -55,10 +55,16 @@ - (void)sensorModuleDidSubscribeForAccelerometerUpdatesOfExperience:experienceId
     [self.manager setAccelerometerUpdateInterval:0.1f];
     [self.manager startAccelerometerUpdatesToQueue:[NSOperationQueue mainQueue] withHandler:^(CMAccelerometerData *data, NSError *error) {
       for (void (^handler)(NSDictionary *) in self.accelerometerHandlers.allValues) {
+        
+        // Inverse values to match Android
```
Comment (ide) [inline ios/Exponent/Kernel/Services/EXSensorManager.m]:
```suggestion
        // Invert values to match Android
```

File: packages/expo-sensors/ios/EXSensors/EXSensorsManager.m [hunk 2]
```diff
@@ -64,10 +64,15 @@ - (void)sensorModuleDidSubscribeForAccelerometerUpdates:(id)scopedSensorModule
       __strong EXSensorsManager *strongSelf = weakSelf;
       if (strongSelf) {
         for (void (^handler)(NSDictionary *) in strongSelf.accelerometerHandlers.allValues) {
+          // Inverse values to match Android
```
Comment (ide) [inline packages/expo-sensors/ios/EXSensors/EXSensorsManager.m]:
```suggestion
          // Invert values to match Android
```

File: packages/expo-sensors/ios/EXSensors/EXSensorsManager.m [hunk 3]
```diff
@@ -64,10 +64,15 @@ - (void)sensorModuleDidSubscribeForAccelerometerUpdates:(id)scopedSensorModule
...
+          double x = -data.acceleration.x;
+          double y = -data.acceleration.y;
+          double z = -data.acceleration.z;
+          
           handler(@{
-                    @"x": [NSNumber numberWithDouble:data.acceleration.x],
-                    @"y": [NSNumber numberWithDouble:data.acceleration.y],
-                    @"z": [NSNumber numberWithDouble:data.acceleration.z]
+                    @"x": [NSNumber numberWithDouble:x],
```
Comment (ide) [inline packages/expo-sensors/ios/EXSensors/EXSensorsManager.m]:
```suggestion
                    @"x": @(x),
```

**Review by ide — CHANGES_REQUESTED** [review]

File: docs/versions/unversioned/sdk/accelerometer.md [hunk 4]
```diff
@@ -2,24 +2,29 @@
 title: Accelerometer
 ---
 
-Access the device accelerometer sensor(s) to respond to changes in
-acceleration in 3d space.
+Access the device accelerometer sensor(s) to respond to changes in rotation in 3d space. Data is measured in _`m/s^2`_.
```
[the remark this answers is not in the document]
Comment (EvanBacon) [reply docs/versions/unversioned/sdk/accelerometer.md]: lmao unless something gets swapped on our side I think they're wrong

File: docs/versions/unversioned/sdk/accelerometer.md [hunk 5]
```diff
@@ -2,24 +2,29 @@
...
+
+This module interfaces directly with native code of the same name. This is why the Accelerometer functions like a Gyroscope and vise-versa.
+
+For detecting the speed/rate your device is moving in, see the `Gyroscope` module.
+
+> As of v33 values have been inverted on iOS to match Android.
```
[the remark this answers is not in the document]
Comment (EvanBacon) [reply docs/versions/unversioned/sdk/accelerometer.md]: So I actually built the gyro wrong on web I'm seeing. I built it like a gyroscope. Should I rename the native modules on the API side or change the web module functionality?

**Pull request closed without merging**